src/HttpClient/HttpClient.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\HttpClient;
+
+use Tempest\Http\Request;
+use Tempest\Http\Response;
+
+interface HttpClient
```

Comment:
@brendt - Do we want to keep this interface or require all modifications to be through the driver?

Reply:
That is an issue with the initializer rework done #184, not this specific PR.

Reply:
I cannot open an issue at the moment, as I know it's still in development, but nothing magical, I'm using Tempest as a package:

Clone/download the repository
Run composer install
Configure the .env file

Reply:
Let's stick with the interface 👍